gateway/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id("spring-conventions")
+    jacoco
+    `java-test-fixtures`
+}
+
+dependencies {
+    implementation(project(":internal-api"))
+    implementation(project(":core"))
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("io.nats:jnats:2.16.14")
+    implementation("io.projectreactor:reactor-core:3.6.10")
+    testImplementation(testFixtures(project(":core")))
```

Comment:
looks like a duplicated dependency. see line 18

Reply:
testImplementation dependency is not visible in testFixtures, w=so i added it